CMakeLists.txt

```diff
@@ -63,6 +63,7 @@ option(ACTS_BUILD_EXAMPLES_EDM4HEP "Build EDM4hep-based code in the examples" OF
 option(ACTS_BUILD_EXAMPLES_EXATRKX "Build the Exa.TrkX example code" OFF)
 option(ACTS_BUILD_EXAMPLES_GEANT4 "Build Geant4-based code in the examples" OFF)
 option(ACTS_BUILD_EXAMPLES_HEPMC3 "Build HepMC3-based code in the examples" OFF)
+option(ACTS_BUILD_HASHING "Build Hashing-based code in the core" OFF)
```

Comment:
I think core components should not be toggled. Either we can build this all the time or it should be a plugin I suppose

Reply:
Moving the code to core comes from a discussion with @noemina allowing it then to be used with Athena.
If it is not an issue to build it all the time, I can remove this option.

Reply:
I think it is great to have it in code but then I would remove the build switch

Reply:
Done